controller_manager:
  ros__parameters:
    update_rate: 100
    use_sim_time: true

    arm_cont:
      type: velocity_controllers/JointGroupVelocityController

    pendulum_broad:
      type: joint_state_broadcaster/JointStateBroadcaster

arm_cont:
  ros__parameters:
    joints:
      - arm_joint

pendulum_broad:
  ros__parameters:
    joints:
      - pendulum_joint